lms/djangoapps/course_api/views.py

```diff
@@ -231,9 +233,10 @@ class CourseListView(DeveloperErrorViewMixin, ListAPIView):
               }
             ]
     """
+    class CourseListPageNumberPagination(NamespacedPageNumberPagination):
+        max_page_size = 100
 
-    pagination_class = NamespacedPageNumberPagination
-    pagination_class.max_page_size = 100
```

Comment:
**Reviewer Note:** This was overriding this class setting for all uses of `NamespacedPageNumberPagination`.